### DISCUSSION TITLE
articles/how-do-i-test-react-components

### DISCUSSION THREAD

**Discussion by giscus** [tiagomichaelsousa/website#38] [Comments]
# articles/how-do-i-test-react-components

Introduction Following the well-known sentence  there is no such thing as a dumb question  I also like to say that 
there is no such thing as  a perfect way to…

https://tiagomichaelsousa.dev/articles/how-do-i-test-react-components

**Comment by comoser** [comment]
Great article Tiago :) definitely something everyone should do at tests in general

**Reply by tiagomichaelsousa** [reply]
Thank you for the feedback @comoser! 🙏

Some of the tips and tricks that were mentioned during the article were constinouly improved when I had the opportunity to work with you! 
So big thanks and #kudos to you as well! 🚀
